api/v1alpha1/hostedcluster_types.go

```diff
@@ -101,7 +101,7 @@ type HostedClusterSpec struct {
 // ServicePublishingStrategyMapping defines the service being published and  metadata about the publishing strategy.
 type ServicePublishingStrategyMapping struct {
 	// Service identifies the type of service being published
-	// +kubebuilder:validation:Enum=APIServer;OAuthServer;OIDC;Konnectivity
+	// +kubebuilder:validation:Enum=APIServer;OAuthServer;OIDC;Konnectivity;Ignition
```

Comment:
this is orthogonal to this PR by as anote for future API refinements I wonder if this validation should live in the ServiceType type definition.